spring-amqp/src/main/java/org/springframework/amqp/core/MessageProperties.java

```diff
@@ -55,11 +55,11 @@
 
 	public static final String X_DELAY = "x-delay";
 
-	static final String DEFAULT_CONTENT_TYPE = CONTENT_TYPE_BYTES;
+	protected static final String DEFAULT_CONTENT_TYPE = CONTENT_TYPE_BYTES;
 
-	static final MessageDeliveryMode DEFAULT_DELIVERY_MODE = MessageDeliveryMode.PERSISTENT;
+	protected static final MessageDeliveryMode DEFAULT_DELIVERY_MODE = MessageDeliveryMode.PERSISTENT;
```

Comment:
M-m-m. Would you mind to explain why did you leave these two constant `protected`, since we had been talking to make them all `public`?